### ISSUE TITLE
Loading spinner graphic plays indefinetly after failing to retrieve graphic   

### ISSUE THREAD

**Issue by kevinmcdonagh** [mttkay/droid-fu#3]
The loading spinner should offer a fallback plan for a default if the graphic cannot be sourced correctly. At the moment I have a list of images and sometimes the graphic fails to load which I can observe from the log but to a user they wrongly will believe that the graphic is still attempting to load.

**Comment by robbypond** [comment]
Agree. Option to include a default image to show on failure would be nice.
